libjlm/include/jlm/tooling/Command.hpp

```diff
@@ -384,6 +384,7 @@ class JlmOptCommand final : public Command {
 
   filepath InputFile_;
   filepath OutputFile_;
+public:
```

Comment:
Please provide an accessor method and keep this private:
`const std::vector<Optimization> &
Optimizations() const noexcept
{
return Optimizations_
}`

Reply:
Fixed